Servers/Administration/OJS.Servers.Administration/Controllers/Api/TestsController.cs

```diff
@@ -27,7 +27,7 @@ public class TestsController : BaseAdminApiController<Test, int, TestsInListMode
         IGridDataService<Test> testGridDataService,
         ITestsBusinessService testsBusinessService,
         TestAdministrationModelValidator validator,
-        IValidator<BaseDeleteValidationModel<int>> deleteValidator,
```

Comment:
Do we have an issue to fix the need for adding empty validator instead of using the generic one?

Reply:
Yes, this is it https://github.com/SoftUni-Internal/exam-systems-issues/issues/1180